.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:/Program Files (x86)/Python36-32/python.exe"
```

Comment:
Oops, that shouldn't have been committed. I thought it had been removed beforehand.